--- a/include/ddns.h
+++ b/include/ddns.h
@@ -39,7 +39,9 @@
 #define SUPPORT_ADDR	PACKAGE_BUGREPORT
 
 /* Test values */
-#define DEFAULT_CONFIG_FILE	"/etc/inadyn.conf"
+#define DEFAULT_CONFIG_FILE	"/Apps/opt/etc/inadyn.conf"
+#undef _PATH_VARRUN
+#define _PATH_VARRUN	"/Apps/opt/var/"
 #define RUNTIME_DATA_DIR        _PATH_VARRUN     "inadyn/"
 #define DEFAULT_PIDFILE         RUNTIME_DATA_DIR "inadyn.pid"
 
